### ISSUE TITLE
ImportError: lxml.html.clean module is now a separate project lxml_html_clean

### ISSUE THREAD

**Issue by JonathanDoughty** [RhetTbull/macnotesapp#45]
Following both`brew install macnotesapp` and `brew upgrade macnotesapp` if I try `notes --help` I get

```
Traceback (most recent call last):
  File "<~>.local/bin/notes", line 5, in <module>
    from macnotesapp.__main__ import cli_main
  File "<~>.local/pipx/venvs/macnotesapp/lib/python3.12/site-packages/macnotesapp/__main__.py", line 3, in <module>
...
    from lxml.html.clean import Cleaner
  File "<~>.local/pipx/venvs/macnotesapp/lib/python3.12/site-packages/lxml/html/clean.py", line 18, in <module>
    raise ImportError(
ImportError: lxml.html.clean module is now a separate project lxml_html_clean.
Install lxml[html_clean] or lxml_html_clean directly.
```

Web searches suggest this results from a [recent spinoff from lxml](https://pypi.org/project/lxml-html-clean/)

> because many reports about possible security vulnerabilities were filed for lxml and that make the project problematic for security-sensitive environments

I'd try to provide a PR but my python lacks poetry and pipx.

**Comment by RhetTbull** [comment]
@all-contributors please add @JonathanDoughty for bug

**Comment by RhetTbull** [comment]
I didn't even know macnotesapp could be installed via brew.  I have tried to replicate this with pip and pipx and it installs fine (it's able to resolve a version of the dependencies that satisfy all constraints).

**Comment by RhetTbull** [comment]
Should be fixed in v0.6.0.

**Comment by JonathanDoughty** [comment]
Thanks!  tl;dr: I was able to get the revised version working by following the README_DEV.md and running `./build.sh` to create `dist/notes.zip` and then extracting the executable from there, after `brew install poetry` and `poetry install; poetry shell` 

`pytest -v -s tests/` also ran through my existing Notes and created the new test ones so at least I know that part works.

I was NOT able to get it running using pipx: I kept getting 0.5.3 and/or running into python version issues -- looks like I may need to install yet another local python  version. This is my first attempt at using pipx so blame it on user error.  

FYI: the 0.6.0 release does not include `notes.zip` whereas the assets for earlier version do. Thanks again!
